packages/core/src/types/model.ts

```diff
@@ -0,0 +1,18 @@
+import * as tf from '@tensorflow/tfjs-node-gpu';
+
+
+export interface PipcookModel {
+  model: any;
+  metrics?: any;
+  predict: any;
+  config?: any;
+}
+
+export interface TfJsLayersModel extends PipcookModel {
+  model: tf.LayersModel;
+}
+
+export interface PytorchModel extends PipcookModel {
+  criterion: any;
+  optimizer: any;
+}
```

Comment:
The above should be moved to corresponding plugins?

Reply:
Pipcook-core can officially provides interface for Tfjs/Tf/Pytorch model definition?

Reply:
It requires the installation for these libs, and it usually takes too much time.

Reply:
this will not install these libraries, this is just for interface

Reply:
See https://github.com/alibaba/pipcook/pull/56/files/60caf30054cbe864aae294dba5c73ee97bd5e7ce#diff-8db3ac627dd924923c375f66ed6364cfR11-R18, we used a type from tfjs, which requires the installation :)

Reply:
BTW, could we move these types for supporting libraries to our Plugin API later?

Reply:
the link does not direct to specific place. Could you point out which file is using tfjs? I actually have removed all dependencies on specific framework

Reply:
https://github.com/alibaba/pipcook/blob/60caf30054cbe864aae294dba5c73ee97bd5e7ce/packages/core/src/types/model.ts

Reply:
this has already removed in this PR branch~